lab_2/main.py

```diff
@@ -92,7 +217,34 @@ def find_diff_in_sentence(original_sentence_tokens: tuple, suspicious_sentence_t
     :param lcs: a longest common subsequence
     :return: a tuple with tuples of indexes
     """
-    pass
+    if not isinstance(original_sentence_tokens, tuple) or not isinstance (suspicious_sentence_tokens, tuple):
+        return ()
+    if not isinstance (lcs, tuple):
+        return ()
+    list_of_tuples = [original_sentence_tokens, suspicious_sentence_tokens]
+    for current_tuple in list_of_tuples:
+        for word in current_tuple:
+            if not isinstance(word, str):
+                return ()
+    for word in lcs:
+        if not isinstance (word, str):
+            return ()
+    list_of_lists = [[], []]
+    list_of_tuples = [original_sentence_tokens, suspicious_sentence_tokens]
+    flag = 0
+    for current_list, current_tuple in zip(list_of_lists, list_of_tuples):
+        for word in current_tuple:
+            if word not in lcs and flag == 0:
```

Comment:
flag = 0 or 1 is definitely better, but how can you simplify the check `flag == 0` or `flag == 1` ?